app/src/test/java/de/tum/in/tumcampusapp/database/dao/CalendarDaoTest.java

```diff
@@ -68,10 +68,11 @@ private CalendarItem createCalendarItem(String status, DateTime startDate, DateT
      */
     @Test
     public void getAllNotCancelledAll() {
-        dao.insert(createCalendarItem("OPEN", DateTime.now()));
-        dao.insert(createCalendarItem("OTHER", DateTime.now()));
-        dao.insert(createCalendarItem("OTHER", DateTime.now()));
-        dao.insert(createCalendarItem("OPEN", DateTime.now()));
+        DateTime now = DateTime.now();
```

Comment:
I'd make this DateTime static, i.e. a hardcoded time, that is known to be ok

Reply:
The issue with test failing was `cleanUp` function that used SQL's localtime time function (and subtracting 3 months from it to compare). Yes, DateTime.now() generates a new time, but with current commit I've isolated each test case to expected times, so that exact behaviour is expected. Hardcoding time would decrease test stability as they're supposed to fail if something not under behaviour happens.

I'll try to investigate what timezone SQL database uses.

Reply:
Hm, seems reasonable.